screwdriver/screwdriver.go

```diff
@@ -270,6 +271,10 @@ func (a api) put(url *url.URL, bodyType string, payload io.Reader) ([]byte, erro
 	return a.write(url, "PUT", bodyType, payload)
 }
 
+func (a api) GetAPIURL() string {
+	return a.baseURL
```

Comment:
Okay, seeing you guys have it here: https://github.com/screwdriver-cd/template-main/blob/46769f6ba2e3956c3d22c07cb5ad9a02d819fda9/index.js#L29.

But shall we just get it from launcher instead of hardcoding the v4 in different places?